src/tests/ftest/deployment/ior_per_rank.yaml

```diff
@@ -46,16 +46,15 @@ container:
     control_method: daos
 ior:
     client_processes:
-          ppn: 16
+          ppn: 32
     test_file: testFile
-    repetitions: 1
     flags: "-w -r -R -C -e -g -D 30 -G 27 -k -Q 1 -v"
     dfs_oclass: 'SX'
     transfer_sizes:
         - 1M
         - 256B
     block_size: 150G
-    write_x: 0.1 # Max 10% performance difference.
+    write_x: 0.15 # Max 15% performance difference.
```

Comment:
As an improvement to this test later, we could verify the performance of each rank is within some margin of all other ranks. I.e.
verify rank0 perf within X% of 22.5G
verify rank1 perf within X% of 22.5G
verify rank0 perf within Y% of rank1 perf